client/leasing-portal/src/App.tsx

```diff
@@ -33,7 +35,9 @@ function App() {
             }
           />
           {/* TO-DO: add route to support page  */}
-          <Route path="/support" element={<Filters />} />
+          <Route path="/support" element={<ApptListing />} />
```

Comment:
Needs to be changed as per mentioned To-Do

Reply:
yeah, will be handled in next sprint